docs/configuration.md

```diff
@@ -30,6 +30,14 @@ by passing the `--local` option to the `config` command.
 poetry config virtualenvs.create false --local
 ```
 
+{{% note %}}
+Your local configuration of Poetry application is stored in `poetry.toml` file,
+which is separate from `pyproject.toml`.
+
+It is recommended you do not check-in this file into your repository
+since it may contain user specific or sensitive information.
+{{% /note %}}
+
```

Comment:
```suggestion
{{% note %}}
Your local configuration of Poetry application is stored in `poetry.toml` file,
which is separate from `pyproject.toml`.
{{% /note %}}

{{% warning %}}
Be mindful about checking in this file into your repository since it may contain user-specific or sensitive information.
{{% /note %}}

```